ans-abstract/abstract.tex

```diff
@@ -154,33 +154,43 @@ \section{Results}
+	\caption{Diesel gallons consumed each day by MTD from July 1, 2018 to June 30, 2019 \cite{mtd_irecords_2019}.}
 	\label{fig:mtdfuel}
 \end{figure}
 
-The UIUC fleet includes 227 passenger vehicles and 275 service vehicles \cite{noauthor_increase_2020}. The calculations consider only 108 vehicles chosen based on their annual mileage. The 108 vehicles combined consume 269 gasoline gallons per day \cite{holcomb_fueling_2015}. Table \ref{tab:h2req} lists the hydrogen required to supply the UIUC fleet. The table also shows the CO$_2$ emitted by both fleets.
+The UIUC fleet includes both passenger and service vehicles \cite{uiuc_institute_for_sustainability_energy_and_environment_increase_2020}. The calculations consider only the portion of the fleet that operates in town and consumes gasoline \cite{uiuc_personnal_communication}. Figure \ref{fig:uiucfuel} presents the daily consumption of unleaded gasoline by the UIUC fleet in a year. The data go from January 1st of 2019 to December 31st of 2019. The fleet also uses diesel and ethanol but in smaller proportions that future analysis will take into account.
+Table \ref{tab:h2req} lists the hydrogen required to supply the UIUC fleet.
 
-\begin{table}[]
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.95\linewidth]{figures/uiuc-unleaded.png}
+	\hfill
+	\caption{Gasoline gallons consumed each day by the UIUC fleet from January 1, 2019 to December 31, 2019 \cite{uiuc_personnal_communication}.}
+	\label{fig:uiucfuel}
+\end{figure}
+
+\begin{table}[H]
 	\centering
     \caption{Hydrogen required and CO$_2$ produced by MTD and UIUC fleets.}
     \label{tab:h2req}
 \begin{tabular}{l|rr}
 \hline
                    & MTD (Diesel)   & UIUC (Gasoline)  \\ \hline
-Average gal/day    & 1,971.8        & 269.0            \\
-kg of H$_2$/day    & 2,228.2        & 269.0            \\
-CO$_2$ (lbs/day)   & 44,129.5       & 5,283.2          \\
-gal/year           & 719,717.6      & 98,185.0         \\
-kg of H$_2$/year   & 813,280.9      & 98,185.0         \\
-CO$_2$ (lbs/year)  & 16,107,279.9   & 1,928,353.4      \\ \hline
+Average gal/day    & 1,971.8        & 251.8            \\
+kg of H$_2$/day    & 2,228.2        & 251.8            \\
+CO$_2$ (lbs/day)   & 44,129.5       & 4,945.3          \\
+gal/year           & 719,717.6      & 91,925.1         \\
+kg of H$_2$/year   & 813,280.9      & 91,925.1         \\
+CO$_2$ (lbs/year)  & 16,107,279.9   & 1,805,408.9      \\ \hline
 \end{tabular}
 \end{table}
 
-Both fleets combined would consume 2240.8 kg/day of H$_2$. A micro-reactor feeding an iodine-sulfur thermochemical cycle would need 10 MWth of power to meet that daily average demand of hydrogen.
+Table \ref{tab:h2req} also shows the CO$_2$ emitted by MTD and UIUC fleets.
+Both fleets combined would consume 2480 kg/day of H$_2$. A micro-reactor feeding an iodine-sulfur thermochemical cycle would need 10 MWth of power to meet that daily average demand of hydrogen.
```

Comment:
I have not been shown any analysis that connects this number of kg/day to the 10 MWth estimate here. It seems to come out of thin air.  If you have arrived at this number through a calculation, great, please include that calculation here. Otherwise, without analysis to back it up, drop this guesstimate.

```suggestion
Combined, these fleets would consume 2480 kg/day of H$_2$. 
```

Reply:
If this is connected to the assumptions from above, where mentoined the NGNP design goals were to demonstrat 50MWth = 500 kg/h, then a 10MWth reactor will give you 100kg/h, which is 2400kg/day... But, that demonstration never happened, and a need of 2480 is closer to 11MWth in any case.

The scale-up numbers from the other paper are also undemonstrated plans. These numbers were never physically achieved.